File: Generic/SpecificOOP.java
import java.lang.*;

class Arithmetic
{
    public int No1;
    public int No2;
    Arithmetic(int a,int b)
    {
        No1 = a;
        No2 = b;
    }
    public int Arithmetic()
    {
        int Ans = 0;
        Ans = No1 + No2;
        return Ans;
    }
}

class SpecificOOP
{
    public static void main(String arg[])
    {
        Arithmetic obj = new Arithmetic();
        int ret = 0;
        ret = obj.Arithmetic(10,11);
        System.out.println("Addition is : "+ret);
    }
}

File: Generic/SpecificOOP.cpp
#include<iostream>

using namespace std;

class Arithmetic
{
    public:
        int No1,No2;

        Arithmetic(int a,int b)
        {
            No1 = a;
            No2 = b;
        }

        int Addition()
        {
            int ans = 0;
            ans = No1 + No2;
            return ans; 
        }

        int Substraction()
        {
            int ans = 0;
            ans = No1 - No2;
            return ans;
        }
};

int main()
{

    Arithmetic obj(10,11);
    int fret =  obj.Addition();
    cout<<"Addition is"<<fret<<"\n";

    fret =  obj.Substraction();
    cout<<"Substration is"<<fret<<"\n";

    return 0;
}